### ISSUE TITLE
A simple quesiton

### ISSUE THREAD

**Issue by xiikjuy** [UKPLab/sentence-transformers#616]
Hi, thanks for the work. I am new for nlp and would like to use your model in the huggingface library to extract sentence-level semantic embeddings. 
So first, I need to pick any model names containing "stsb" right? 
And this is what I did for example:
```
from transformers import AutoTokenizer, AutoModel
model_name = "sentence-transformers/roberta-large-nli-stsb-mean-tokens"
model = AutoModel.from_pretrained(model_name)
tokenizer = AutoTokenizer.from_pretrained(model_name)
sentence = "Yangliuqing is a market town in Xiqing District."
import torch
emd = model(torch.tensor([tokenizer.encode(sentence)]))[1]
emd.size() # torch.Size([1, 1024])
```
So can I say ```emd``` is a sentence-level semantic embedding ready for use? (my corpus is wiki+books)
And can I change the feature dim 1024?
Thanks!

**Comment by nreimers** [comment]
Have a look here:
https://www.sbert.net/examples/training/distillation/README.html#dimensionality-reduction

You can train a PCA model with the desired number of dimension and apply it on your embeddings to reduce the dim. size.
